src/Packagist/WebBundle/Resources/public/css/main.css

```diff
@@ -1734,3 +1734,7 @@ svg.chart {
 .icon-lock:before { content: '🔒'; } /* '\1f512' */
 .icon-lock-open:before { content: '🔓'; } /* '\1f513' */
 .icon-rocket:before { content: '🚀'; } /* '\1f680' */
+
+.markdown-body details summary {
```

Comment:
I even suggest to use only `details summary` (or maybe even `summary` if that's enough specificity). I don't think the extra specificity to restrict it to `.markdown-body` is necessary.

Reply:
I think it's a good idea to leave `.markdown-body`, this can give the context about why the rule is there. Do you agree?

Also, I have reviewed `web/css/github/markdown.css` and this have have not been updated since 2016, although the latest version from this css file is from February 2020. Don't you think the best thing to do is to update this file? I can do in this PR or in a new one.

Reply:
updating it would indeed be a good idea (and would solve this issue as the new version include this rule)